docs/basic_usage.md

```diff
@@ -22,7 +22,8 @@ $  surfactant generate [OPTIONS] CONFIG_FILE SBOM_OUTFILE [INPUT_SBOM]
 **--recorded_institution**: (optional) the name of the institution collecting the SBOM data (default: LLNL)\
 **--output_format**: (optional) changes the output format for the SBOM (given as full module name of a surfactant plugin implementing the `write_sbom` hook)\
 **--input_format**: (optional) specifies the format of the input SBOM if one is being used (default: cytrics) (given as full module name of a surfactant plugin implementing the `read_sbom` hook)\
-**--help**: (optional) show the help message and exit
+**--help**: (optional) show the help message and exit\
+**--inclulde_all_files**: Include all files in the SBOM, rather than just those recognized by Surfactant
```

Comment:
```suggestion
**--include_all_files**: (optional) include all files in the SBOM, rather than just those recognized by Surfactant
```

Just a stray character, and altering format some to match other entries (maybe including `(optional)` for all the flags is redundant).